components/buttons/editButton.js

```diff
@@ -0,0 +1,18 @@
+import Image from "next/image";
+import editButton from "../../public/edit_button.svg";
+import styled from "styled-components";
+
+export default function EditButton() {
+  return (
+    <ToPosition>
+      <Image src={editButton} width={60} height={60} />
+    </ToPosition>
+  );
+}
+
+const ToPosition = styled.div`
```

Comment:
Could this be a `button` element instead of a `div`. The component is named `Editbutton`.